src/GovUk.Education.ExploreEducationStatistics.Content.Model/Repository/ReleaseDataFileRepository.cs

```diff
@@ -15,7 +15,7 @@ public class ReleaseDataFileRepository : IReleaseDataFileRepository
     {
         private readonly ContentDbContext _contentDbContext;
 
-        private static readonly List<FileType> SupportedFileTypes = new()
+        private static readonly List<FileType> SupportedFileTypes = new ()
```

Comment:
accidental whitespace here

Reply:
I believe this is the way Rider normally formats this - rather than having no space. In the codebase, we sometimes have a space and sometimes don't, so I'm going to leave this.